nucliadb/Makefile

```diff
@@ -47,6 +47,7 @@ test-%:
 
 test-cov-%:
 	$(PYTEST) \
+		-x \
```

Comment:
I don't like this here since in CI these tests run in parallel and are then retried serially. We expect failures in the parallel phase, so putting this here means that the first test to fail in parallel will make it to immediately go to serial mode and will make tests slower.

Reply:
Oh true, I'll remove it